exercises/standard_library_types/iterators5.rs

```diff
@@ -0,0 +1,61 @@
+// iterators5.rs
+
+pub fn fibonacci(i: u64) -> u64 {
+    // Complete this function to return the ith fibonacci number
+    // Do not use:
+    // - return
+    // For extra fun don't use:
```

Comment:
Let's say "challenge" instead of "fun" here?

Reply:
Ha! This is a copy and paste job of the previous exercise (iterators4). I'm happy to change it; shall I change both?

Actually, I think I to rethink the wording altogether, it fails to recognise the similarity with the previous exercise and doesn't flow very well, leading to a bad user experience...

Reply:
That would be great!